libs/indicore/indicom.c

```diff
@@ -331,7 +335,7 @@ double time_ns()
     ts.tv_sec = mts.tv_sec;
     ts.tv_nsec = mts.tv_nsec;
 #else
-    timespec_get(&ts, TIME_UTC);
+    #error "Unsupported platform"
```

Comment:
Build fail for Windows CI:
```
D:\a\indi\indi\libs\indicore\indicom.c(338,1): fatal  error C1189: #error:  "Unsupported platform" [D:\a\indi\indi\build\libs\indicore\indicore.vcxproj]
  indidevapi.c
```

Reply:
```cmake
CHECK_FUNCTION_EXISTS(timespec_get HAVE_TIMESPEC_GET)
```
the result is a false negative. The function exists. It's possible that in `Visual Studio 17` the function is inline, either define or requires a system library.

Reply:
Works:
```cmake
include(CheckSymbolExists)
check_symbol_exists(timespec_get time.h HAVE_TIMESPEC_GET)
```

Reply:
OK. I didn't read through till CheckFunctionExists module footnote. After reading it, I seems to me a good idea to replace every 'check_function_exists' to 'check_symbol_exists'.